Makefile.inc1

```diff
@@ -2349,14 +2340,13 @@ real-sign-packages:	_pkgbootstrap .PHONY
 .if ${PKG_BIN_VERSION} < 11700
 	printf "packing_format = \"${PKG_FORMAT}\";\n" >> ${WSTAGEDIR}/meta
 .endif
-	@[ -L "${REPODIR}/${PKG_ABI}/latest" ] && unlink ${REPODIR}/${PKG_ABI}/latest; \
```

Comment:
Likewise, this hunk to use a forced `ln -s` to remove the previous link can be its own commit I think.